run_simulation.py

```diff
@@ -52,16 +54,18 @@
                     help='length of the latent space features')
 parser.add_argument('-ve', '--vae_epochs', default=60,
                     help='how many epochs for VAE models')
-parser.add_argument('-ae', '--adage_epochs', default=100,
+parser.add_argument('-ae', '--adage_epochs', default=60,
                     help='how many epochs for ADAGE models')
 parser.add_argument('-b', '--batch_size', default=30,
                     help='How many samples to use to update weights')
 parser.add_argument('-r', '--vae_learning_rate', default=0.0005,
                     help='learning rate for the VAE models')
-parser.add_argument('-a', '--adage_learning_rate', default=0.0005,
+parser.add_argument('-a', '--adage_learning_rate', default=0.0004,
                     help='learning Rate for the ADAGE models')
 parser.add_argument('-i', '--adage_noise', default=0.1,
                     help='noise corruption for model')
+parser.add_argument('-w', '--adage_untied_weights', action='store_false',
```

Comment:
Shouldn't this be labeled `adage_tied_weights` instead of untied weights?

Reply:
Ack I just got what was going on here. Feel free to ignore this comment.

Reply:
Yeah, it is a bit confusing. The default ADAGE option should be `tied_weights`. I would want a user to specify if they want to use untied weights.